CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 ### 0.16.1 (Next)
 
 * Your contribution here.
+* [#359](https://github.com/slack-ruby/slack-ruby-client/pull/359) Refactor server error handling to account for non-JSON 500 errors. - [@agrobbin](https://github.com/agrobbin).
```

Comment:
This is actually a fix, so I would say "Handle non-JSON 500 errors"

Missing a colon, `:` and the period should be at the end, see danger wanrings.